src/index.js

```diff
@@ -70,8 +71,27 @@ listItemsContainer.addEventListener('keypress', (e) => {
       }
       const num = e.target.getAttribute('data-para');
       tasksList[num].description = e.target.textContent;
-      window.location.reload();
     }
   }
   return localStorage.setItem('list', JSON.stringify(tasksList));
-});
\ No newline at end of file
+});
+
+const tickmarked = document.querySelectorAll('.li-check');
+tickmarked.forEach((list) => {
+  list.addEventListener('change', (e) => {
+    if (e.currentTarget.checked === true) {
+      list.nextElementSibling.classList.add('overlined');
+      const idnx = e.currentTarget.getAttribute('data-id');
+      tasksList[idnx].completed = true;
+    } else {
+      list.nextElementSibling.classList.remove('overlined');
+      const idnx = e.currentTarget.getAttribute('data-id');
+      tasksList[idnx].completed = false;
+    }
+    localStorage.setItem('list', JSON.stringify(tasksList));
+    window.location.reload();
```

Comment:
[OPTIONAL] Well done on the project overall 💯 but it would be great if you find a way to make it work without reloading the screen after each interaction. It doesn't give the best user experience, would be great to fix but the most important thing is that your project is working. Congrats 🎊

Reply:
Thank you for your option. I did notice it. However, I will fix it in my leisure time.